src/ai/backend/manager/config/local.py

```diff
@@ -0,0 +1,935 @@
+        Controls how long the manager can hold a lock while checking if a session can be created.
+        Should be balanced to prevent both deadlocks and race conditions.
+        """,
+        examples=[30.0, 60.0],
+    )
+    session_start_lock_lifetime: float = Field(
+        default=30,
+        description="""
+        Maximum lifetime in seconds for session start locks.
+        Controls how long the manager can hold a lock while starting a session.
+        Longer values are safer but may block other managers longer on failure.
+        """,
+        examples=[30.0, 60.0],
+    )
+    pid_file: FilePath = Field(
+        default=Path(os.devnull),
+        description="""
+        Path to the file where the manager process ID will be written.
+        Useful for service management and monitoring.
+        Set to /dev/null by default to disable this feature.
+        """,
+        examples=["/var/run/manager.pid"],
+    )
+    allowed_plugins: Optional[List[str]] = Field(
+        None,
+        description="""
+        List of explicitly allowed plugins to load.
+        If specified, only these plugins will be loaded, even if others are installed.
+        Useful for controlling exactly which plugins are active.
+        Leave as None to load all available plugins except those in disabled_plugins.
+        """,
+        examples=[["example.plugin.what.you.want"]],
+    )
+    disabled_plugins: Optional[List[str]] = Field(
+        default=None,
+        description="""
+        List of plugins to explicitly disable.
+        These plugins won't be loaded even if they're installed.
+        Useful for disabling problematic or unwanted plugins without uninstalling them.
+        """,
+        examples=[["example.plugin.what.you.want"]],
+    )
+    hide_agents: bool = Field(
+        default=False,
+        description="""
+        Whether to hide detailed agent information in API responses.
+        When enabled, agent details are obscured in user-facing APIs.
+        Useful for security in multi-tenant environments.
+        """,
+        examples=[True, False],
+    )
+    agent_selection_resource_priority: List[str] = Field(
+        default=["cuda", "rocm", "tpu", "cpu", "mem"],
+        description="""
+        Priority order for resources when selecting agents for compute sessions.
+        Determines which resources are considered more important during scheduling.
+        Default prioritizes GPU resources (CUDA, ROCm) over CPU and memory.
+        """,
+        examples=[["cuda", "rocm", "tpu", "cpu", "mem"]],
+    )
+    importer_image: str = Field(
+        default="lablup/importer:manylinux2010",
+        description="""
+        Container image used for the importer service.
+        The importer handles tasks like installing additional packages.
+        Should be compatible with the Backend.AI environment.
+        """,
+        examples=["lablup/importer:manylinux2010"],
+    )
```

Comment:
`importer_image` is not used anywhere.
It is probably removed config.
It would be good to explicitly indicate this in the description.